content/resources/chef_client_cron/_index.md

```diff
@@ -30,7 +30,7 @@ syntax_full_code_block: |-
     environment              Hash
     hour                     Integer, String # default value: "*"
     job_name                 String # default value: "chef-client"
-    log_directory            String # default value: /Library/Logs/Chef on macOS and /var/log/chef otherwise
+    log_directory            String
```

Comment:
Was this intentional? Seems kinda bad.

Reply:
If they were super wide they get kicked off this top section. The long defaults still land below in the details section.